index.html

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>Products App</header>
+    <section id="product-selection">Pick A Product</section>
```

Comment:
```suggestion
    <div id="root">Pick A Product</div>
```
With the new class component architecture, we want to be appending to a root div with the dom generated from each components template, starting with survey-app.js as the parent.